docs/src/references/c8y-configuration-management.md

```diff
@@ -225,6 +230,8 @@ When there is no retain message the plugin sends an error to the cloud on an eve
   * C8Y config plugin: publishs the config file content as MQTT retain message to `configs/bar.conf` 
   * external device `child1`: recognizes the MQTT message on `configs/bar.conf` and processes the new config file content
 
+--------------------------------------------------------------------------------
+
 **Proposal 2:**
```

Comment:
Having proper section titles would be better. Both markdown and wording can be improved.

```suggestion
### Deliver and update configuration files over HTTP
```

Reply:
Improved now, see https://github.com/cstoidner/thin-edge.io/commit/0e2f15737d6652ba84aebf475bf710d3d560099f